tests/user.test.ts

```diff
@@ -1,35 +1,54 @@
 import { JsAuthenticator, User } from 'tonomy-id-sdk';
-import { APIClient } from '@greymass/eosio';
+import { api } from './services/eosio';
 
-test('Create a new account', async () => {
+beforeEach((): void => {
+    jest.setTimeout(60000);
+});
+
+test('Create a new ID of a person', async () => {
     const auth = new JsAuthenticator();
 
     const user = new User(auth);
 
-    user.savePassword("myPassword123!");
-    user.savePIN("4568");
-    user.saveFingerprint();
+    // user.savePassword("myPassword123!");
+    // user.savePIN("4568");
+    // user.saveFingerprint();
+    // user.saveLocal();
+
+    await user.createPerson("jack2233");
 
-    await user.createPerson("jack");
-    const accountName = user.accountName.toString();
+    const accountName = user.accountName;
 
-    const eosioClient = new APIClient({ url: "http://localhost:8888" });
-    const accountInfo = await eosioClient.v1.chain.get_account(accountName);
+    // for this call to work:
```

Comment:
right now, to run the integration test you need to follow these instructions as there is a bug in the@greymasss/eosio package